packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Localize } from '@deriv/translations';
+import { WS } from 'Services/ws-methods';
+import { connect } from 'Stores/connect';
+
+// check market in coming 7 days
+const days_to_check_before_exit = 7;
+
+const getTradingTimes = async target_time => {
+    return new Promise(resolve => {
+        WS.tradingTimes(target_time).then(data => {
+            if (data.error) resolve({ api_initial_load_error: data.error.message });
+            resolve(data);
+        });
+    });
+};
+
+const getSymbol = (target_symbol, trading_times) => {
+    let symbol;
+    const { markets } = trading_times;
+    for (let i = 0; i < markets.length; i++) {
+        const { submarkets } = markets[i];
+        for (let j = 0; j < submarkets.length; j++) {
+            const { symbols } = submarkets[j];
+            symbol = symbols.find(item => item.symbol === target_symbol);
+            if (symbol !== undefined) break;
+        }
+        if (symbol !== undefined) break;
+    }
+    return symbol;
+};
+
+const whenMarketOpens = async (days_offset, target_symbol) => {
+    // days_offset is 0 for today, 1 for tomorrow, etc.
+    if (days_offset > days_to_check_before_exit) return undefined;
+    const now = new Date();
+    let when_market_opens;
+    const date_target = new Date(now);
```

Comment:
`now` is not used anywhere else. Can we remove line 38 and just use `new Date()` here?